### PR TITLE
localize slider labels

### PR DESCRIPTION
**Pull request by owenca**

### CODE REVIEW

File: source/Makefile [hunk 1]
```diff
@@ -87,7 +87,7 @@ OPTIMIZE :=
 # 	will recreate only the "locales/en.catkeys" file. Use it as a template
 # 	for creating catkeys for other languages. All localization files must be
 # 	placed in the "locales" subdirectory.
-LOCALES = 
+LOCALES = en de es fr ja
```
Comment (waddlesplash) [inline source/Makefile]: You don't have any of these locales in the tree...?

Comment (owenca) [reply source/Makefile]: "make catkeys" creates locales/en.catkeys. Is it what you want?

Comment (pulkomandy) [reply source/Makefile]: You should not add any languages there until the application has actually been translated. People translating it will provide catkeys files for the other languages, and once we have those, we can add them here (and in the git repository).

Comment (owenca) [reply source/Makefile]: Thanks for explaining it! So should I just leave LOCALES empty as it was?

**Comment by owenca** [comment]
just talked to humdinger

File: source/DemoView.cpp [hunk 2]
```diff
@@ -48,13 +48,8 @@ DemoView::AttachedToWindow()
 void
 DemoView::Draw(BRect updateRect)
 {
-	//demo background
-//	SetHighColor(255, 255, 255, 255);
-//	FillRoundRect(updateRect, 2, 2);
-	SetHighColor(0, 0, 0, 255);
```
Comment (pulkomandy) [inline source/DemoView.cpp:54]: This looks unrelated to localization. Please explain the changes :)

File: source/DemoView.cpp [hunk 3]
```diff
@@ -65,9 +60,6 @@ DemoView::Draw(BRect updateRect)
 		fRadius += Settings.lwidth / 2 + Settings.lspace;
 		fColor.alpha += fAlphaMod;
 	}
-
-	ConstrainClippingRegion(NULL);
```
Comment (pulkomandy) [inline source/DemoView.cpp:69]: same here.

Comment (owenca) [reply source/DemoView.cpp:69]: After reading the BeBook link you gave me, I learned that drawing attributes are reset to their default values at the beginning of the Draw() function. I also tested it to make sure. So there is no need to reset background color and clipping region explicitly.

**Comment by owenca** [comment]
Also moved the initialization of the clipping region and its center from the constructor to Draw() because the size of the demo area is not finalized until Draw() is called.
